packages/teams-js/src/public/mail.ts

```diff
@@ -63,6 +64,25 @@ export function composeMail(composeMailParams: ComposeMailParams): Promise<void>
   });
 }
 
+/**
+ * Compose a new email in the user's mailbox, opening it in the drafts UX instead of the standard email.
```

Comment:
Any new function we add should be marked with the @beta tag so that developers know that the function may change or be broken with little or no warning. There should be other examples in the library you can check.